worker/pkg/workflows/datasync/activities/gen-benthos-configs/processors.go

```diff
@@ -366,21 +366,12 @@ func constructBenthosJsProcessor(jsFunctions, benthosOutputs []string) string {
 
 	jsCode := fmt.Sprintf(`
 (() => {
-function setNestedProperty(obj, path, value) {
-	path.split('.').reduce((current, part, index, parts) => {
-		if (index === parts.length - 1) {
-			current[part] = value;
-		} else {
-			current[part] = current[part] ?? {};
-		}
-		return current[part];
-	}, obj);
-}
 %s
 const input = benthos.v0_msg_as_structured();
```

Comment:
Just checking but we still need the `input` right because we pass it in to the transformer functions.

Reply:
yep still need the input